ms365_accountcreator/translations/de/LC_MESSAGES/messages.po

```diff
@@ -90,6 +96,10 @@ msgstr "Schritt 1"
 msgid "Your st****** email address"
 msgstr "Deine st****** E-Mail Adresse"
 
+#: ms365_accountcreator/templates/index.html:49
+msgid "st******@stud.uni-stuttgart.de"
+msgstr ""
```

Comment:
Translation still missing especially in the line below this.

Reply:
I think for this one it is ok. And the one below is actually not missing, just babel wanted to add an empty string and then the rest of the translation in the next lines

Reply:
ok, forgot to expand the diff view